CONTRIBUTING.md

```diff
@@ -14,17 +29,9 @@ For any feature requests, please use the [issue tracker](https://github.com/sage
 
 For features that you have a proposed solution for, please write an [RFC](./rfcs/README.md).
 
-## Pull Request Guidelines
-
-* Before submitting a pull request, check the [issue tracker](https://github.com/sage/carbon/issues) to see if the feature or bug has already been discussed. If it has, check with us before beginning work on it to avoid duplicated effort.
-
-* If no issue has been raised, please raise one first before beginning work. You can then link back to this when submitting a pull request to address it.
-
-* Create a branch from master and work in the `src` folder.
+## Automated issue management
 
-* Include a detailed description of the feature you are adding in the PR. Ensure you have included information on how to setup and QA your new feature or bug-fix.
-
-* We prefer detailed commit messages and squashed commits. PRs with commit messages like 'WIP' and 'addressed comments', or long lists of commits will likely require amending."
+We use a bot to help us manage issues. The bot automatically labels issues where there has been no activity for 365 days as `stale`. We then consider the issue for closure. 
 
 ## Project Structure
 
```

Comment:
We should remove the `Project Structure` section. It's likely to become outdated (like it is now).